### PR TITLE
add vi/docs for Service

### PR DESCRIPTION
**Pull request by ititandev**
Add docs for Service in Vietnamese.

Signed-off-by: Le Duc Manh ldmanhit@gmail.com

### CODE REVIEW

File: content/vi/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -0,0 +1,873 @@
...
+
+ * Có một lịch sử dài hiện thực DNS không tuân thủ record TTLs, và caching kết quả truy vấn sau khi chúng hết hạn.
+ * Một số ứng dựng chỉ truy vấn DNS một lần và lưu trữ kết quả vô thời hạn.
+ * Ngay cả khi ứng dụng và thư viện thực hiện phân giải đúng cách, TTL thấp hoặc bằng không có thể gây ra tải cao cho DNS mà sau đó trở nên khó quản lý.
+
+### Chế độ user space proxy {#proxy-mode-userspace}
+
+Trong chế độ này, kube-proxy theo dõi Kubernetes master để thêm và gỡ bỏ đối tượng Service và Endpoint. Mỗi Service sẽ mở một port (chọn ngẫu nhiên) trên local node. Bất kỳ kết nối nào tới "proxy port" này được chuyển tiếp đến một trong các Pod backend của Service (được báo cáo qua Endpoint). kube-proxy lấy cấu hình `SessionAffinity` của Service để tính toán khi quyết định backend Pod nào được sử dụng.
+
+Cuối cùng, user-space proxy cài đặt quy tắc iptables mà tiếp nhận lưu lượng tới `clusterIP` (ảo) và `port` của Service. Những quy tắc này chuyển hướng traffic tới proxy port để chuyển hướng tới backend Pod.
```
Comment (truongnh1992) [inline content/vi/docs/concepts/services-networking/service.md]: Should use: `iptables rules` instead of `quy tắc iptables`

**Comment by truongnh1992** [comment]
@ititandev Please remove all of untranslated file from your PR.

**Comment by ititandev** [comment]
@truongnh1992 What should I do next?

File: content/vi/docs/templates/feature-state-alpha.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+This feature is currently in a *alpha* state, meaning:
+
+* The version names contain alpha (e.g. v1alpha1).
+* Might be buggy. Enabling the feature may expose bugs. Disabled by default.
+* Support for feature may be dropped at any time without notice.
+* The API may change in incompatible ways in a later software release without notice.
+* Recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of long-term support.
+
```
Comment (sftim) [inline content/vi/docs/templates/feature-state-alpha.txt:8]: Is it feasible to translate this to Vietnamese?

File: content/vi/docs/templates/feature-state-beta.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+This feature is currently in a *beta* state, meaning:
+
+* The version names contain beta (e.g. v2beta3).
+* Code is well tested. Enabling the feature is considered safe. Enabled by default.
+* Support for the overall feature will not be dropped, though details may change.
+* The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release. When this happens, we will provide instructions for migrating to the next version. This may require deleting, editing, and re-creating API objects. The editing process may require some thought. This may require downtime for applications that rely on the feature.
+* Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases. If you have multiple clusters that can be upgraded independently, you may be able to relax this restriction.
+* **Please do try our beta features and give feedback on them! After they exit beta, it may not be practical for us to make more changes.**
```
Comment (sftim) [inline content/vi/docs/templates/feature-state-beta.txt:8]: Is it feasible to translate this to Vietnamese?

File: content/vi/docs/templates/feature-state-deprecated.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+This feature is *deprecated*. For more information on this state, see the [Kubernetes Deprecation Policy](/docs/reference/deprecation-policy/).
```
Comment (sftim) [inline content/vi/docs/templates/feature-state-deprecated.txt:2]: Is it feasible to translate this to Vietnamese?

File: content/vi/docs/templates/feature-state-stable.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+This feature is *stable*, meaning:
+
+* The version name is vX where X is an integer.
+* Stable versions of features will appear in released software for many subsequent versions.
```
Comment (sftim) [inline content/vi/docs/templates/feature-state-stable.txt:5]: Is it feasible to translate this to Vietnamese?

File: content/vi/docs/templates/index.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+headless: true
+
+resources:
+- src: "*alpha*"
+  title: "alpha"
+- src: "*beta*"
+  title: "beta"
+- src: "*deprecated*"
+  title: "deprecated"
```
Comment (sftim) [inline content/vi/docs/templates/index.md:10]: _ngừng sử dụng_ ?

**Comment by kbhawkey** [comment]
Hello @ititandev .
As this pull request has been inactive for some time, I am closing it.
You can reopen this PR at any time to continue to work on the changes.
/close

**Pull request closed without merging**